Added NextDNS (#215)
riverar/wsa-app-compatibility: README.md

| Nettfart Mobile | 3.6.8 | 12 | ✅ | The app must be given network permissions in App Settings
| Network IP Scanner | 3.2 | 11 | ⚠️ | Only scans WSA's own VirtWifi network
| NewPipe | 0.22.1 | 11 | ✅

> | NextDNS | 1.2 | 12 | ✅ || the VPN workaround should be used, only once after installation, to allow the app to create VPN connections.

Reviewer: ```suggestion
| NextDNS | 1.2 | 12 | ✅ || VPN workaround is needed once after installation, to allow the app to create VPN connections.
```
